### ISSUE TITLE
float not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#423]
```
What steps will reproduce the problem?
1.create a html file and paste the attached demo_html.html file content in that 
html file which you have created.

What is the expected output? What do you see instead?
divs suppose to appear side by side but in the pdf tey are appearing in 
different row.

What version of dompdf are you using? What version of PHP? On what
operating system?
we are using 0.6.1 version of dompdf.

Please provide the HTML source code you want to convert, or any additional
information.
I have attached a screenshot of how it is coming after creating pdf. 

```

Original issue reported on code.google.com by `AmitTiwa...@gmail.com` on 14 Feb 2012 at 1:29
- Merged into: #19

Attachments:
- [post_dompdf.jpg](https://storage.googleapis.com/google-code-attachments/dompdf/issue-423/comment-0/post_dompdf.jpg)
- [demo_html.html](https://storage.googleapis.com/google-code-attachments/dompdf/issue-423/comment-0/demo_html.html)

**Comment by GoogleCodeExporter** [comment]
```
Thank you to report any problem related to float in the appropriate issue.
```

Original comment by `fabien.menager` on 14 Feb 2012 at 2:24
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
FYI, float support is available starting with dompdf 0.6.0 beta 3. It is still 
in the "experimental" stage. As such you have to manually enable it in 
dompdf_config.inc.php or dompdf_config.custom.inc.php.

config ref: 
http://code.google.com/p/dompdf/source/browse/tags/dompdf_0-6-0_beta3/dompdf/dom
pdf_config.inc.php#301

render ref: 
http://eclecticgeek.com/dompdf/debug.php?identifier=149e4d6cb74787c9e829c4f77556
0914
```

Original comment by `eclecticgeek` on 14 Feb 2012 at 3:10

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:16
- Added labels: **Restrict-AddIssueComment-Commit**
